device-types/Cisco/N9K-C9408.yaml

```diff
@@ -0,0 +1,49 @@
+---
+manufacturer: Cisco
+model: N9K-C9408
+slug: cisco-n9k-c9408
+part_number: N9K-C9408
+u_height: 4
+is_full_depth: false
+comments: '[Cisco Nexus 9400 Series Switches Data Sheet](https://www.cisco.com/c/en/us/products/collateral/switches/nexus-9000-series-switches/nexus9400-series-switches-ds.html'
+power-ports:
+  - name: PS1
+    type: iec-60320-c20
+    maximum_draw: 2000
+  - name: PS2
+    type: iec-60320-c20
+    maximum_draw: 2000
+  - name: PS3
+    type: iec-60320-c20
+    maximum_draw: 2000
+  - name: PS4
+    type: iec-60320-c20
+    maximum_draw: 2000
+module-bays:
```

Comment:
These chassis have removable PSU so please model them as 'module-bays' and then add the required PSU modules